game.py

```diff
@@ -12,16 +12,31 @@
     '-----------'
 ]
 
-
 def snowman(snowman_word):
     """Complete the snowman function
     replace "pass" below with your own code
     It should print 'Congratulations, you win!'
     If the player wins and, 
     'Sorry, you lose! The word was {snowman_word}' if the player loses
     """
-    pass
+    correct_letter_guess_statuses = build_letter_status_dict(snowman_word)
+    wrong_guesses_list = []
+    while(len(wrong_guesses_list) < SNOWMAN_MAX_WRONG_GUESSES and not is_word_guessed(snowman_word, correct_letter_guess_statuses)):
```

Comment:
I might consider adding a white line between lines 23 and 24 to help visually/logically separate the different parts of code. Lines 22-23 declare variables and line 24 is the start of a while loop. This is an opinion and you might prefer not to add it!

The style guide says "Use blank lines in functions, sparingly, to indicate logical sections."

You can read more about [blank lines](https://peps.python.org/pep-0008/#blank-lines) in code as described by the Python Style Guide.

```suggestion
    correct_letter_guess_statuses = build_letter_status_dict(snowman_word)
    wrong_guesses_list = []
    
    while(len(wrong_guesses_list) < SNOWMAN_MAX_WRONG_GUESSES and not is_word_guessed(snowman_word, correct_letter_guess_statuses)):
```